### ISSUE TITLE
build script assumes a valid GOPATH checkout

### ISSUE THREAD

**Issue by anthonator** [coreos/dex#439]
I'm having trouble building v0.4.0 with a script that worked fine for v0.3.0. Looking at the release notes for v0.4.0 and I'm not seeing anything that would cause incompatibility.

Here's my script:

```
curl -L https://github.com/coreos/dex/archive/v0.4.0.tar.gz | tar zx
cp -R dex-v0.4.0/* /opt/dex
rm -rf dex-v0.4.0

cd /opt/dex
./build
```

Results in error:

```
can't load package: package github.com/coreos/dex/cmd/dex-worker: cannot find package "github.com/coreos/dex/cmd/dex-worker" in any of:
	/usr/local/go/src/github.com/coreos/dex/cmd/dex-worker (from $GOROOT)
	/go/src/github.com/coreos/dex/cmd/dex-worker (from $GOPATH)
can't load package: package .: no buildable Go source files in /opt/dex
```

Same script works fine for 0.3.0.

**Comment by ericchiang** [comment]
Ah, I believe we stared assuming you have downloaded dex to a valid GOPATH.

**Comment by anthonator** [comment]
Gotcha. Is this a bug or documentation issue?

**Comment by ericchiang** [comment]
Not really a bug but something we should fix (am working on it).

**Comment by anthonator** [comment]
This change to my script fixed the issue:

```
curl -L https://github.com/coreos/dex/archive/v0.4.0.tar.gz | tar zx
cp -R dex-v0.4.0/* /go/src/github.com/coreos/dex
rm -rf dex-v0.4.0

cd /go/src/github.com/coreos/dex
./build
```

**Comment by anthonator** [comment]
👍 thanks @ericchiang

**Comment by whereisaaron** [comment]
@ericchiang's fix for this, the 'env' script in #440 has been removed now?
